lib/orderbook/errors.ts

```diff
@@ -13,7 +13,9 @@ const errorCodes = {
   LOCAL_ID_DOES_NOT_EXIST: codesPrefix.concat('.9'),
   QUANTITY_DOES_NOT_MATCH: codesPrefix.concat('.10'),
   CURRENCY_MISSING_ETHEREUM_CONTRACT_ADDRESS: codesPrefix.concat('.11'),
-  INSUFFICIENT_OUTBOUND_BALANCE: codesPrefix.concat('.12'),
+  SWAP_CLIENT_NOT_FOUND: codesPrefix.concat('.12'),
```

Comment:
I believe the tests are failing here because it's changing the error code for an existing error, `INSUFFICIENT_OUTBOUND_BALANCE` which impacts the tests. This should just create two new errors and assign them the 13 and 14 error codes without modifying existing errors.